pallets/msa/src/lib.rs

```diff
@@ -848,11 +876,61 @@ pub mod pallet {
 					Self::deposit_event(Event::MsaRetired { msa_id });
 				},
 				None => {
-					log_err!("SignedExtension did not catch invalid MSA for account {:?}, ", who);
+					log::error!(
+						"SignedExtension did not catch invalid MSA for account {:?}, ",
+						who
+					);
 				},
 			}
 			Ok(Some(T::WeightInfo::retire_msa(num_deletions)).into())
 		}
+
+		/// Request to be a provider.  Creates a proposal for council approval to create a provider from a MSA
+		///
+		/// # Errors
+		/// - [`NoKeyExists`](Error::NoKeyExists) - If there is not MSA for `origin`.
+
+		#[pallet::call_index(11)]
+		#[pallet::weight(T::WeightInfo::request_to_be_provider(provider_name.len() as u32))]
+		pub fn request_to_be_provider(
```

Comment:
maybe propose_provider since it submits a proposal?

Reply:
It's now `propose_to_be_provider() `